bazel/emscripten_toolchain/link_wrapper.py

```diff
@@ -25,20 +26,8 @@
 param_filename = sys.argv[1][1:]
 param_file_args = [l.strip() for l in open(param_filename, 'r').readlines()]
 
-output_index = param_file_args.index('-o') + 1
-orig_output = js_output = param_file_args[output_index]
-outdir = os.path.dirname(orig_output)
-
-# google3-only(TODO(b/139440956): Default to False once the bug is fixed)
-replace_response_file = any(' ' in a for a in param_file_args)
-
-if not os.path.splitext(orig_output)[1]:
-  js_output = orig_output + '.js'
-  param_file_args[output_index] = js_output
-  replace_response_file = True
-
 # Re-write response file if needed.
-if replace_response_file:
```

Comment:
(unrelated) I wonder if we can completely remove this at some point... how does bazel + clang deal with this issue?